MastodonSDK/Sources/MastodonCore/Service/API/APIService+Account.swift

```diff
@@ -199,3 +200,43 @@ extension APIService {
         return response
     }   // end func
 }
+
+extension APIService {
+    public func fetchUser(username: String, domain: String, authenticationBox: MastodonAuthenticationBox)
```

Comment:
We can't use `accountLookup()` as it is now because it does not create a Persistent record. In case a record doesn't exist, we need to create one upon lookup. So I'm adding a new function `accountSearch()` that does the lookup and creates a record.